gui/default/syncthing/core/syncthingController.js

```diff
@@ -745,6 +791,16 @@ angular.module('syncthing.core')
             ]);
         }
 
+        function refreshWebauthnState() {
+            return $http.get(urlbase + '/webauthn/state').success(function (data) {
+                $scope.webauthn.uneditedState = deepCopy(data);
+                if (!$scope.webauthn.state) {
+                    $scope.webauthn.state = deepCopy(data);
+                }
+                console.log("refreshWebauthnState", data);
```

Comment:
Debugging leftover?  We probably shouldn't keep this.

Reply:
Fixed in 86e9193685e9c523c6c6d163c24e3def17afaa20 (part of https://github.com/emlun/syncthing/pull/7) and 065c303d8c4f89579133060a2043a390495c46ba.